fishtest/fishtest/templates/tests_view.mak

```diff
@@ -209,3 +217,24 @@ Gaussian Kernel Smoother&nbsp;&nbsp;<div class="btn-group"><button id="btn_smoot
   %endfor
  </tbody>
 </table>
+
+<script>
+  $(function() {
+    var apiUrlBaseMaster = "https://api.github.com/repos/official-stockfish/Stockfish";
+    $.ajax({
+      url: apiUrlBaseMaster + "/compare/master...${run['args']['resolved_base'][:7]}",
+      headers: {
+        Accept: "application/vnd.github.v3.diff"
+      },
+      success: function(response) {
+        var $masterDiff = $("#master-diff");
+        if (response.length === 0) {
```

Comment:
is  `response.length === 0` (3x =) correct syntax ?

Reply:
Yes, it's equal without type coercion.